.github/CODEOWNERS

```diff
@@ -1,3 +1 @@
-# Define code owners (individuals or teams that are responsible for code in this repository)
-# More about code owners at https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
 *       @ConduitIO/conduit-core
```

Comment:
So the @conduitio/conduit-core doesn't have access, we should probably use @conduitio-labs/conduit-core here.